core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -747,17 +751,22 @@ class HttpGoogleServicesDAO(
   }
 
   override def getGenomicsOperation(opId: String): Future[Option[JsObject]] = {
-    implicit val service = GoogleInstrumentedService.Genomics
-    val genomicsServiceAccountCredential = getGenomicsServiceAccountCredential
-    genomicsServiceAccountCredential.refreshToken()
 
-    if (opId.startsWith("operations")) {
+
+    def papiv1Handler(opId: String) = {
       // PAPIv1 ids start with "operations". We have to use a direct http call instead of a client library because
       // the client lib does not support PAPIv1 and PAPIv2 concurrently.
+      val genomicsServiceAccountCredential = getGenomicsServiceAccountCredential
+      genomicsServiceAccountCredential.refreshToken()
       new GenomicsV1DAO().getOperation(opId, OAuth2BearerToken(genomicsServiceAccountCredential.getAccessToken))
-    } else {
+    }
```

Comment:
does this still need a v1 handler?

Reply:
Probably not... but it feels like a call I'd rather make separately tbh, just in case querying still works even though starting new workflows doesn't.